### PR TITLE
Integrated Gemini API with Worker

### PR DESCRIPTION
**Pull request by samuzzal-choudhury**

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -190,35 +191,34 @@ def get_info(cls, search_key):
...
-    return True
+def server_run_flow(flow_name, flow_args):
+	"""Run a flow.
 
+	:param flow_name: name of flow to be run as stated in YAML config file
+	:param flow_args: arguments for the flow
+	:return: dispatcher ID handling flow
+	"""
+	logger.info('Running flow {}'.format(flow_name))
+	print('Running flow {}'.format(flow_name))
```
Comment (miteshvp) [inline src/utils.py]: redundant debug statement

File: src/utils.py [hunk 2]
```diff
@@ -190,35 +191,34 @@ def get_info(cls, search_key):
...
+	dispacher_id = run_flow(flow_name, flow_args)
 
-    def __init__(self):
-        """Initialize class.
-
-        Intializes selinon for async functionality.
-        """
-        init_selinon()
+	elapsed_seconds = (datetime.datetime.now() - start).total_seconds()
+	print("It took {t} seconds to start {f} flow.".format(
```
Comment (miteshvp) [inline src/utils.py]: redundant debug statements

File: src/utils.py [hunk 3]
```diff
@@ -190,35 +191,34 @@ def get_info(cls, search_key):
...
-        """
-        current_app.logger.debug('Running flow {}'.format(flow_name))
-        start = datetime.datetime.now()
+def scan_repo(data):
+    """Scan function."""
+    args = {'github_repo': data['git_url'],
+            'github_sha': data['git_sha'],
+            'email_ids': data['email_ids']}
+    d_id = server_run_flow('osioAnalysisFlow', args)
+    print ("DISPATCHER ID = {}".format(d_id))
```
Comment (miteshvp) [inline src/utils.py]: maybe use logger here as well

**Comment by samuzzal-choudhury** [comment]
[test]

**Comment by GeetikaBatra** [comment]
@IMHO, we need to add worker in docker-compos.yaml, as tests use local environment and this code change makes use of worker, I suppose. Please correct me if I am wrong.

**Comment by samuzzal-choudhury** [comment]
@GeetikaBatra the `runtests.sh` does install worker before running the tests. To make things clear, `worker` is used as a library in the gemini server.

**Comment by samuzzal-choudhury** [comment]
Merging this as the PR has been merged and the build ran successfully.

**Pull request merged**